ui/src/routes/guide.svelte

```diff
@@ -0,0 +1,141 @@
+
+		<h3>Regional selection</h3>
+		<p>It is possible to select NUTS1 regions to inherently select its contained NUTS2 region. This is particularly useful because NUTS1 regions don't vary over the years whereas NUTS2 regions do.</p>
+		<p>By default all NUTS2 regions are selected. The "globe" button toggles the menu where you can select the NUTS1 regions.</p>
+		<p>The selected NUTS2 regions can be highlighted or filtered.</p>
+		<p>Highlighting is useful to focus the attention on the trends for the desired regions while still being able to see them all.</p>
+		<p>Filtering is useful to isolate trends for the desired regions: when filtering, the color scale on the left gets updated only considering the values of the active trends.</p>
+
+		<h3>indicator info and download</h3>
+		<p>Clicking in the "i" icon at the right of the page title shows an info panel reporting the indicator metadata (data provenance, date of processing, etc).</p>
+		<p>In that same panel there is a button to download the <a href={csvWikiURL}>CSV file</a> of the current indicator.</p>
+		<p>The panel cak be dismissed by clicking iot the page background.</p>
+
+		<h2>Indicator yearly data</h2>
+
+		<p>This page cak be reached either by clicking on a dot in the index page or by clicking on one of the dots of the timeline in the trend of a certain indicator.<p></p>
+
+		<h3>Sidebar</h3>
+		<p>The sidebar has the same function described above: you can use it to navigate indicators trend page.<p></p>
+
+		<h3>Map</h3>
+		<p>On the left there is a choropleth showing the indicator values on the map of UK NUTS2 regions.</p>
+		<p>Gray regions indicate that there is no data available for that region.</p>
+		<p>The map shows cities with the most research intensive universities (called "Trac peer group A"), as used previously on maps within BEIS because they are key to BEIS regional policies.</p>
+
+		<h3>Barchart</h3>
+		<p>On the rightleft there is a barchart showing the disrtibution of the indicator values in various NUTS2 regions, sorted by value in descendent order.</p>
+
+		<h3>Regional selection</h3>
+		<p>As for the trends view, it ir porrible toi highlight or filter the NUTS2 regions for a certain year by selecting a NUTS1 region and using the switch to change selection mode.</p>
+		<p>When filtering the map zooms on the currently selected regions.</p>
+
+		<h3>Indicator info and download</h3>
+		<p>The "i" icon at the right of the page title has the very same function described above.</p>
+
+		<h2>Downloading all the indicators at once</h2>
+		<p>Please click on "download" icon on the top-right of the website header to download a zip file containing the CSV files of all all the indicators.</p>
+		<p>That zip file also contains an extra CSV file (titled <code>beis_indicators_{version}.csv</code>) containing all datapoints of all indicators in a single place.</p>
```

Comment:
```suggestion
		<p>That zip file also contains an extra CSV file (titled <code>beis_indicators_{version}.csv</code>) containing all datapoints of all indicators in a single file.</p>
```
Is this what you meant or did you mean for a single geographic region?